nilearn/_utils/numpy_conversions.py

```diff
@@ -157,7 +158,7 @@ def csv_to_array(csv_path, delimiters=" \t,;", **kwargs):
         An array containing the data loaded from the CSV file.
     """
     csv_path = stringify_path(csv_path)
-    if not isinstance(csv_path, str):
+    if not isinstance(csv_path, (str, pathlib.Path)):
```

Comment:
need to think if that makes sense as stringify_path will always return a str if passed a Path...